op-chain-ops/upgrades/l1.go

```diff
@@ -234,7 +235,26 @@ func L1StandardBridge(batch *safe.Batch, implementations superchain.Implementati
 		return err
 	}
 
-	calldata, err := l1StandardBridgeABI.Pack("initialize", superchainConfigProxy)
+	var messenger, otherBridge common.Address
+	if config != nil {
```

Comment:
Prefer to always pull from remote and compare values to config if the config is present and error if there is a mismatch

Reply:
agreed, but wouldn't this deviate from the existing strategy in the same file for other contracts?

Reply:
We should do this strategy for all of the contracts